lib/bolero-generator/src/prelude.rs

```diff
@@ -1,5 +1,5 @@
 pub use crate::{
-    constant, gen, gen_with,
```

Comment:
we'll need to keep `gen` and `gen_with` in the prelude so it doesn't break existing code

Reply:
Oh, will take a look at that.